### PR TITLE
add GDC info metric

### PR DESCRIPTION
**Pull request by tomkralidis**
Adds info metrics on GDC init.

cc @golfvert

### CODE REVIEW

File: metrics/gdc.csv [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Name,Labels,Description,Child,Initialize
+wmo_wis2_gdc_metrics_info,,"Diagnostic of GDC centre identifier, API URL, and GB subscribed to",-,true
```
Comment (golfvert) [inline metrics/gdc.csv:2]: add labels: centre_id|instance|subscribed_to|url

**Review by golfvert — CHANGES_REQUESTED** [review]

File: metrics/gdc.csv (hunk 1 above)
Comment (tomkralidis) [reply metrics/gdc.csv:2]: The above is supposed to be a "info" metric (https://prometheus.github.io/client_python/instrumenting/info/), which prometheus / OpenMetrics allows for.

Thinking more, if we consider this (in the WIS2 context) as "not a metric", I could simply add this as an endpoint somewhere else in GDC.  The question is then whether a given global service should only emit WIS2 metrics as a rule.

**Comment by kaiwirt** [comment]
In my opinion a global service can also omit other metrics as well. I would then however recommend not to use wmo_wis2_gdc_metrics_info but maybe gdc_metrics_info or wmo_gdc_metrics_info to seperate the metrics for WIS2 from other metrics

**Review by kaiwirt — COMMENTED** [review]
See my comment on the naming of the metrics

**Comment by tomkralidis** [comment]
Then the question becomes whether this is considered a valuable metric for a GDC to emit to WIS2 monitor.  If yes, the PR stands, else we can close.

**Comment by antje-s** [comment]
Perhaps splitting this metric into:
1) wmo_wis2_gdc_gb_connected_flag - label: centre_id with value of GB subscribed to
(as this part is a valuable metric for global monitoring in my opinion)
and one info metric. 
I'm not so sure about the info metric, as the pure values as a string for API and gdc centre_id would be static and therefore not really monitoring metrics. But you could, for example, do an internal function test for the API and then deliver  for example
2) wmo_wis2_gdc_api_status - label= API URL and value 0/1 (=down/up)
With the info metric, I'm not so sure  how deeply you want to test the functionality of the API for a regular metric (e.g. just one test request or testing all functions). But such a metric for service availability would certainly be useful

**Comment by antje-s** [comment]
Update: 1. already included in metric "wmo_wis2_gdc_connected_flag" and label {centre_id =~ ".*global-broker"}

**Comment by tomkralidis** [comment]
@kaiwirt FYI PR updated to `wmo_wis2_gdc_api_status`.  As @antje-s mentions, `wmo_wis2_gdc_connected_flag` already supported.

**Pull request merged**